src/routes.ts

```diff
@@ -34,5 +35,6 @@ export const reactPageRoutes = [
     exact: true,
   },
   { path: '/dont-panic/hedvig', Component: LazyDontPanic, exact: true },
+  { path: '/referrals/:code', Component: Referral, exact: true },
```

Comment:
we need to remember to add this to CF but awesome 🙌

Reply:
Absolutely! 🙂